modules/jetstream-maxtext-deployment/templates/custom-metrics-stackdriver-adapter/hpa.jetstream.yaml.tftpl

```diff
@@ -0,0 +1,30 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: jetstream-hpa
+  namespace: default
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: maxengine-server
+  minReplicas: ${hpa_min_replicas}
+  maxReplicas: ${hpa_max_replicas}
+  metrics:
+%{ if length(regexall("jetstream_.*", hpa_type)) > 0 }
+  - type: Pods
+    pods:
+      metric:
+        name: prometheus.googleapis.com|${hpa_type}|gauge
+      target:
+        type: AverageValue
+        averageValue: ${hpa_averagevalue_target}
+%{ else }
+  - type: External
+    external:
+      metric:
+        name: kubernetes.io|node|accelerator|memory_used
```

Comment:
I see in the prometheus adapter it has the equivalent External name parameterized. Should this one for CMSA also be?